commit 16187430f6c1348b8b7cd279fec834db564a965c
Author: P33M <p33m@github.com>
Date:   Wed Apr 24 14:25:09 2019 +0100

    Revert "cgroup: Disable cgroup "memory" by default"
    
    This reverts commit cd6ce4d0ded13c94ff5208c679ed5e030263149b.

diff --git a/kernel/cgroup/cgroup.c b/kernel/cgroup/cgroup.c
index 3beaab75b081..63dae7e0ccae 100644
--- a/kernel/cgroup/cgroup.c
+++ b/kernel/cgroup/cgroup.c
@@ -5290,8 +5290,6 @@ int __init cgroup_init_early(void)
 }
 
 static u16 cgroup_disable_mask __initdata;
-static u16 cgroup_enable_mask __initdata;
-static int __init cgroup_disable(char *str);
 
 /**
  * cgroup_init - cgroup initialization
@@ -5332,12 +5330,6 @@ int __init cgroup_init(void)
 
 	mutex_unlock(&cgroup_mutex);
 
-	/* Apply an implicit disable... */
-	cgroup_disable("memory");
-
-	/* ...knowing that an explicit enable will override it. */
-	cgroup_disable_mask &= ~cgroup_enable_mask;
-
 	for_each_subsys(ss, ssid) {
 		if (ss->early_init) {
 			struct cgroup_subsys_state *css =
@@ -5721,28 +5713,6 @@ static int __init cgroup_disable(char *str)
 }
 __setup("cgroup_disable=", cgroup_disable);
 
-static int __init cgroup_enable(char *str)
-{
-	struct cgroup_subsys *ss;
-	char *token;
-	int i;
-
-	while ((token = strsep(&str, ",")) != NULL) {
-		if (!*token)
-			continue;
-
-		for_each_subsys(ss, i) {
-			if (strcmp(token, ss->name) &&
-			    strcmp(token, ss->legacy_name))
-				continue;
-
-			cgroup_enable_mask |= 1 << i;
-		}
-	}
-	return 1;
-}
-__setup("cgroup_enable=", cgroup_enable);
-
 /**
  * css_tryget_online_from_dir - get corresponding css from a cgroup dentry
  * @dentry: directory dentry of interest
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1434_16187430f6c1348b8b7cd279fec834db564a965c.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1434_16187430f6c1348b8b7cd279fec834db564a965c.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
